src/core/auth/auth.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+angular.module('core.auth').factory('Auth', ['$http', 'Environment', 'Session', 'Users',
+  function ($http, Environment, Session, Users) {
+    const baseUrlApi = Environment.wBooksApiUrl + 'users/sessions';
+    return {
+      authenticate: (session) => {
+        return $http.post(baseUrlApi, session).then(async resp => {
+          if (resp.data.access_token) {
+            Session.create(resp.data);
+            $http.defaults.headers.common.Authorization = resp.data.access_token;
+            let user = await Users.information();
+            if (!user) {
+              user = {
```

Comment:
why this mocked values? move to a constant aoutside

Reply:
They have already been removed.